modules/persistent-storage-csi-manila-dynamic-provisioning.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-manila.adoc
+
+[id="persistent-storage-csi-manila-dynamic-provisioning_{context}"]
+= Dynamically provisioning Manila CSI volumes
+
+{product-title} installs a default StorageClass.
+
+The YAML files that are created are completely decoupled from Manila and from its Container Storage Interface (CSI) plug-in. As an application developer, you can dynamically provision ReadWriteMany (RWX) storage and deploy Pods with applications that safely consume the storage using YAML manifests.
+
+You can use the same Pod and persistent volume claim (PVC) definitions on-premise that you use with {product-title} on AWS, GCP, Azure, and other platforms, with the exception of the storage class reference in the PVC definition.
+
+.Prerequisites
+
+* OpenStack is deployed with appropriate Manila share infrastructure so that it can be used to dynamically provision and mount volumes in {product-title}.
+
```

Comment:
There is another prerequisite, that all OCP compute nodes should be connected to the storage network to be able to mount volumes from this network. 
Actually, it's an obvious prerequisite, I don't know if other platforms mention it in their documentation.

Reply:
We don't mention it elsewhere and our guidance is not to include obvious prereqs, so I'll leave it out.